R/utils.R

```diff
@@ -85,3 +85,18 @@ diag_inverse <- function(x) {
 approx_equal <- function(a, b, tol = sqrt(.Machine$double.eps)) {
   abs(a - b) < tol
 }
+
+#' @title Check the length of an input value
+#' @param x a [numeric] object
+#' @param lng a [numeric] object
+#' @param type a [character] string specifying required typeof
+#' @param fixit a [logical] value, if TRUE force length to lng
+#' @return a [logical] value
```

Comment:
It should return the same type as the input `x`, not a logical value, right?

Reply:
yes.

Reply:
I'm going to merge and fix this in the next pull request.